src/test/java/com/wikia/webdriver/testcases/mercurytests/old/PortableInfoboxTests.java

```diff
@@ -96,18 +94,16 @@ public void mercury_infobox_externalLinkRedirectsToCorrespondingUrl() {
     infobox.isExternalLinkLabelInURL(externalLinkName, externalURL);
   }
 
-  @Test(groups = "mercury_infobox_imagesAndVideosOpenInLightbox", enabled = false)
-  @RelatedIssue(issueID = "XW-1200", comment = "functionality is broken")
+  @Test(groups = "mercury_infobox_imagesAndVideosOpenInLightbox")
   public void mercury_infobox_imagesAndVideosOpenInLightbox() {
     init();
     navigate.toPage(MercurySubpages.INFOBOX_1);
 
     // Check image
     infobox
-        .clickHeroImage()
+        .clickOnImageInInfobox()
         .isLightboxOpened()
-        .closeLightbox()
-        .isTitleOverImageVisible();
```

Comment:
I would feel better if there would be a way to verify if lightbox is close so the test would fail with proper message in that case, but it looks fine for now.